src/ui/components/ShowMoreCard/index.js

```diff
@@ -54,7 +56,11 @@ export class ShowMoreCardBase extends React.Component {
         </div>
         <a className="ShowMoreCard-revealMoreLink" href="#show-more"
           onClick={this.expandText}>
-          {i18n.gettext('Show more info…')}
+          <span className="visually-hidden">
+            {i18n.gettext('Expand to')}
+          </span>
+
+          {i18n.gettext('Read more…')}
```

Comment:
I don't see an ellipsis in the mocks and I'm not a huge fan of them. I think "Read more" can stand on its own.